contracts/AnchoredView/AnchoredView.sol

```diff
@@ -45,14 +45,13 @@ contract AnchoredView is SymbolConfiguration {
     /// @notice The Open Oracle Price Data contract
     OpenOraclePriceData public immutable priceData;
 
-    /// @notice The highest ratio of the new median price to the anchor price that will still trigger the median price to be updated
+    /// @dev The highest ratio of the new median price to the anchor price that will still trigger the median price to be updated
     uint immutable upperBoundAnchorRatio;
 
-    /// @notice The lowest ratio of the new median price to the anchor price that will still trigger the median price to be updated
+    /// @dev The lowest ratio of the new median price to the anchor price that will still trigger the median price to be updated
     uint immutable lowerBoundAnchorRatio;
 
-    /// @notice Average blocks per day, for checking anchor staleness
-    /// @dev 1 day / 15
+    /// @dev Average blocks per day, for checking anchor staleness (1 day / 15)
     uint constant blocksInADay = 5760;
```

Comment:
can delete this now I think ( or change message if it is used for something else )

Reply:
This was in AnchoredView